source/elements/oneDAL/source/index.rst

```diff
@@ -25,10 +25,10 @@ For general information, visit |github|_.
    :maxdepth: 3
 
    introduction.rst
-   terminology.rst
    programming_model/index.rst
    common_interface/index.rst
    data_management/index.rst
    algorithms/index.rst
+   glossary.rst
```

Comment:
This logic of the nested-index and the index on the higher level is a bit unclear to me: if the short dal intro refers to the common glossary whose location does not change whether I get all specs or just DAL spec, why do we need that logic?
Or, we want to change the overall structure of oneAPI specs depending what is chosen, DAL or all?

Reply:
> Or, we want to change the overall structure of oneAPI specs depending what is chosen, DAL or all?

That's it. The glossary needs to be nested inside the DAL chapter for the standalone DAL spec. When we build full spec, we use nested-index.rst. When we build DAL standalone, we use index.rst

Reply:
okay, it is related to the discussion above: say, I install DAL specs and it would contain say VPL specific terminology; if we choose option 3 above it would help avoid it-we will have common glossary together with dal specific glossary, if it is technically possible